### PR TITLE
chore: Improve articles about Git repositories URLs usage

### PR DESCRIPTION
**Pull request by tolusha**
## What does this pull request change?
Improve articles about Git repositories URLs usage

## What issues does this pull request fix or reference?
https://github.com/eclipse/che/issues/22356

## Specify the version of the product this pull request applies to
main

## Pull Request checklist

The author and the reviewers validate the content of this pull request with the following checklist, in addition to the [automated tests](code_review_checklist.adoc).

- Any procedure:
  - [x] Successfully tested.
- Any page or link rename:
  - [ ] The page contains a redirection for the previous URL.
  - Propagate the URL change in:
    - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/packages/dashboard-frontend/src/services/bootstrap/branding.constant.ts)
- [ ] Builds on [Eclipse Che hosted by Red Hat](https://workspaces.openshift.com).
- [ ] the *`Validate language on files added or modified`* step reports no vale warnings.

### CODE REVIEW

File: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc [hunk 1]
```diff
@@ -59,38 +59,32 @@ pass:c,a,q[{prod-url}]#__<devfile_url>__?__<optional_parameters>__ <1>
...
-* In Mozilla Firefox, go to *☰* *>* *Bookmarks* *>* *Manage bookmarks Ctrl+Shift+O* *>* *Bookmarks Toolbar* *>* *Organize* *>* *Add bookmark*.
-
-* In Google Chrome, go to *⋮* *>* *Bookmarks* *>* *Bookmark manager* *>* *Bookmarks bar* *>* *⋮* *>* *Add new bookmark*.
-====
 [IMPORTANT]
 ====
-To initiate a clone of the Git repository in the filesystem of the new workspace, the devfile must contain project info.
-
-See https://devfile.io/docs/2.2.0/adding-projects.
+The automated git credential injection is not supported.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc]:
```suggestion
Automated Git credential injection is not supported.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 2]
```diff
@@ -7,16 +7,22 @@
 [id="starting-a-new-workspace-with-a-clone-of-a-git-repository"]
 = Starting a new workspace with a clone of a Git repository
 
-Working with {prod-short} in your browser involves multiple URLs:
+With {prod-short}, you can visit a URL in your browser to start
+a new workspace that contains a clone of a Git repository.
+This way, you can clone a Git repository that is hosted on GitHub,
+a GitLab, a Bitbucket or Azure DevOps server instances.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
GitLab, Bitbucket, or Microsoft Azure DevOps server instance.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 3]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
...
 ====
 
-With GitHub and GitLab, you can even use the URL of a specific branch of the repository to be cloned:
-
-* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
-* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
-* `pass:c,a,q[{prod-url}#https://github.com/__<user_or_org>__/__<repository>__/pull/__<pull_request_id>__]` starts a new workspace with a clone of the branch of the pull request.
+* `pass:c,a,q[{prod-url}#https://__<gitlab_host>__/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://__<gitlab_host>__/__<user_or_org>__/__<repository>__/-/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#git@__<gitlab_host>__:__<user_or_org>__/__<repository>__.git]` starts a new workspace from GIT+SSH URL.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
* `pass:c,a,q[{prod-url}#git@__<gitlab_host>__:__<user_or_org>__/__<repository>__.git]` starts a new workspace from Git+SSH URL.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 4]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
...
-* `pass:c,a,q[{prod-url}#https://<bb-host>/users/<user-slug>/repos/<repo-name>/]` starts a new workspace with a clone of the default branch, if a repository was created under the user profile.
-* `pass:c,a,q[{prod-url}#https://<bb-host>/users/<user-slug>/repos/<repo-name>/browse?at=refs%2Fheads%2F<branch-name>]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#https://__<bb_host>__/scm/__<project-key>__/__<repository>__.git]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://__<bb_host>__/users/__<user_slug>__/repos/__<repository>__/]` starts a new workspace with a clone of the default branch, if a repository was created under the user profile.
+* `pass:c,a,q[{prod-url}#https://__<bb_host>__/users/__<user-slug>__/repos/__<repository>__/browse?at=refs%2Fheads%2F__<branch-name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#git@__<bb_host>__:__<user_slug>__/__<repository>__.git]` starts a new workspace from GIT+SSH URL.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
* `pass:c,a,q[{prod-url}#git@__<bb_host>__:__<user_slug>__/__<repository>__.git]` starts a new workspace from Git+SSH URL.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 5]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
...
 
 ====
++
+.The URL syntax for starting a new workspace with a clone of an Azure DevOps Git repository
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
.The URL syntax for starting a new workspace with a clone of a Microsoft Azure DevOps Git repository
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 6]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
...
+====
+* `pass:c,a,q[{prod-url}#https://__<organization>__@dev.azure.com/__<organization>__/__<project>__/_git/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://__<organization>__@dev.azure.com/__<organization>__/__<project>__/_git/__<repository>__?version=GB__<branch>__]` starts a new workspace with a clone of the specific branch.
+* `pass:c,a,q[{prod-url}#git@ssh.dev.azure.com:v3/__<organization>__/__<project>__/__<repository>__]` starts a new workspace from GIT+SSH URL.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
* `pass:c,a,q[{prod-url}#git@ssh.dev.azure.com:v3/__<organization>__/__<project>__/__<repository>__]` starts a new workspace from Git+SSH URL.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 7]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
...
 
+* `pass:c,a,q[{prod-url}#https://__<github_host>__/__<user_or_org>__/__<repository>__]` starts a new workspace with a clone of the default branch.
+* `pass:c,a,q[{prod-url}#https://__<github_host>__/__<user_or_org>__/__<repository>__/tree/__<branch_name>__]` starts a new workspace with a clone of the specified branch.
+* `pass:c,a,q[{prod-url}#https://__<github_host>__/__<user_or_org>__/__<repository>__/pull/__<pull_request_id>__]` starts a new workspace with a clone of the branch of the pull request.
+* `pass:c,a,q[{prod-url}#git@__<github_host>__:__<user_or_org>__/__<repository>__.git]` starts a new workspace from GIT+SSH URL.
 ====
+
 +
-.The URL syntax for starting a new workspace with a clone of a GitHub-hosted repository
+.The URL syntax for starting a new workspace with a clone of a Gitlab instance repository
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
.The URL syntax for starting a new workspace with a clone of a GitLab instance repository
```

File: modules/end-user-guide/pages/mounting-git-configuration.adoc [hunk 8]
```diff
@@ -0,0 +1,54 @@
+:_content-type: PROCEDURE
+:description: Mounting Git configuration
+:keywords: Git, config
+:navtitle: Mounting Git configuration
+:page-aliases:
+
+[id="mounting-git-configuration"]
+= Mounting Git configuration
+
+This procedure describes how to mount a Git config file to a workspace.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/mounting-git-configuration.adoc]:
```suggestion
Follow the instructions below to mount a Git config file in a workspace.
```

File: modules/end-user-guide/pages/mounting-git-configuration.adoc [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    controller.devfile.io/watch-configmap: 'true'
+  annotations:
+    controller.devfile.io/mount-as: subpath
+    controller.devfile.io/mount-path: /etc/
+data:
+  gitconfig: "[user] \n  name = __<git_user_name>__ \n  email = __<git_user_email>__" <2>
+----
++
+<1> A user namespace. Visit `pass:c,a,q[{prod-url}]/api/kubernetes/namespace` to get your {prod-short} user namespace as `name`.
+<2> Your Git user's name and email.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/mounting-git-configuration.adoc]:
```suggestion
<2> Your Git user name and email.
```

File: modules/end-user-guide/pages/mounting-git-configuration.adoc [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+----
+$ {orch-cli} apply -f - <<EOF
+__<ConfigMap_prepared_in_step_1>__
+EOF
+----
+
+.Verification
+
+. xref:starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc[Start a new workspace by using the URL of a remote Git repository] that the Git provider hosts.
+. Once the workspace is started, open a new terminal in the `tools` container and run `git config --get-regexp user.*`. You should observe your Git user's name and email in the output.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/mounting-git-configuration.adoc]:
```suggestion
. Once the workspace is started, open a new terminal in the `tools` container and run `git config --get-regexp user.*`. Your Git user name and email should appear in the output.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc [hunk 11]
```diff
@@ -7,18 +7,18 @@
 [id="starting-a-new-workspace-from-a-devfile-url"]
 = Starting a new workspace from a devfile URL
 
-Working with {prod-short} in your browser involves multiple URLs:
+[IMPORTANT]
+====
+To initiate a clone of the Git repository in the filesystem of the new workspace, the devfile must contain project info.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc]:
```suggestion
To initiate a clone of the Git repository in the filesystem of a new workspace, the devfile must contain project info.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc [hunk 12]
```diff
@@ -7,18 +7,18 @@
...
-* The URLs of your workspaces in use
+See https://devfile.io/docs/2.2.0/adding-projects.
+====
 
 With {prod-short}, you can open a *devfile* URL in your browser to start a new workspace.
 
 pass:[<!-- vale RedHat.Spelling = NO -->]
 
-TIP: You can also use the *Git Repo URL ** field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.
+TIP: You can also use the *Git Repo URL* field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc]:
```suggestion
TIP: You can use the *Git Repo URL* field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a *devfile* to start a new workspace.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc [hunk 13]
```diff
@@ -59,38 +59,32 @@ pass:c,a,q[{prod-url}]#__<devfile_url>__?__<optional_parameters>__ <1>
...
-
-* In Google Chrome, go to *⋮* *>* *Bookmarks* *>* *Bookmark manager* *>* *Bookmarks bar* *>* *⋮* *>* *Add new bookmark*.
-====
 [IMPORTANT]
 ====
-To initiate a clone of the Git repository in the filesystem of the new workspace, the devfile must contain project info.
-
-See https://devfile.io/docs/2.2.0/adding-projects.
+The automated git credential injection is not supported.
+To configure the git credentials, use xref:using-a-git-provider-access-token.adoc[configure personal access token] guide.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc]:
```suggestion
To configure the Git credentials, use the xref:using-a-git-provider-access-token.adoc[configure personal access token] guide.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc [hunk 14]
```diff
@@ -59,38 +59,32 @@ pass:c,a,q[{prod-url}]#__<devfile_url>__?__<optional_parameters>__ <1>
...
 ====
++
+.Verification
+
+After you enter the URL to start a new workspace in a browser tab, it renders the workspace-starting page. When the new workspace is ready, the workspace IDE loads in the browser tab.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-from-a-devfile-url.adoc]:
```suggestion
After you enter the URL to start a new workspace in a browser tab, the workspace starting page appears. When the new workspace is ready, the workspace IDE loads in the browser tab.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 15]
```diff
@@ -7,16 +7,22 @@
 [id="starting-a-new-workspace-with-a-clone-of-a-git-repository"]
 = Starting a new workspace with a clone of a Git repository
 
-Working with {prod-short} in your browser involves multiple URLs:
+With {prod-short}, you can visit a URL in your browser to start
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
With {prod-short}, you can use a URL in your browser to start
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 16]
```diff
@@ -7,16 +7,22 @@
...
-* The URL of your organization's {prod-short} instance, used as part of all the following URLs
-* The URL of the *Workspaces* page of your {prod-short} dashboard with the workspace control panel
-* [.underline]#The URLs for starting a new workspace#
-* The URLs of your workspaces in use
+TIP: You can also use the *Git Repo URL* field on the *Create Workspace* page of your {prod-short} dashboard to enter the URL of a Git repository to start a new workspace.
 
-With {prod-short}, you can visit a URL in your browser to start a new workspace that contains a clone of a Git repository. This way, you can clone a Git repository that is hosted on GitHub, a GitLab instance, or a Bitbucket server.
+[IMPORTANT]
+====
+Starting a workspace using an HTTPS URL from a Git service other mentioned above will fail.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
Starting a workspace using an HTTPS URL from a Git service other than GitHub, GiLab, Bitbucket, or Microsoft Azure DevOps will fail.
```

File: modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc [hunk 17]
```diff
@@ -51,50 +57,68 @@ pass:c,a,q[{prod-url}]#__<git_repository_url>__?__<optional_parameters>__ <1>
 <1> See xref:optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc[].
 ====
 +
+[TIP]
+====
+It is possible to use GIT+SSH URLs to start a new workspace.
```
Comment (deerskindoll) [inline modules/end-user-guide/pages/starting-a-new-workspace-with-a-clone-of-a-git-repository.adoc]:
```suggestion
You can use GIT+SSH URLs to start a new workspace.
```

**Review by deerskindoll — APPROVED** [review]
lgtm

**Comment by themr0c** [comment]
I hereby publicly confirm that running the "Validate conversion to plain asciidoc" step is a leftover dating from the time we didn't use the Antora PDF extension to generate a monolithic AsciiDoc file for the downstream build, and that I recommend to remove from the repository the `tools/antora-to-plain-asciidoc.sh` and the associated `.vale/styles/PlainAsciiDoc` Vale style.

**Comment by tolusha** [comment]
@deerskindoll 
Please review my latest changes.

**Review by deerskindoll — APPROVED** [review]
lgtm

**Review by ibuziuk — APPROVED** [review]
:+1:

**Pull request merged**